docs/optimization_models.rst

```diff
@@ -1,5 +1,610 @@
+implemented. This approach is explained in the following sections. It totally makes sense to have
+some experiences with *oemof-solph* to understand this toolbox more easily.
+
+Scope
+-----
+
+The following questions can be addressed using the `optimize_investment` method
+of the *ThermalNetwork*:
+
+* What is the cost-optimal topology and dimensioning of a DHS piping system, 
+  given the locations of potential central heat supply plants, the potential
+  locations for the DHS piping system (e.g. street network),
+  and the position of consumers?
+* In addition to the first question, what is the cost-optimal expansion
+  of a given DHS system?
+* Is it cost-efficient to build a DHS at all, if there a consumer-wise heat
+  supply alternatives? (Comparison of central and de-central supply strategies)
+* What is the optimal dispatch of the heat producers? (In case there are no
+  expansion options, but just existing DHS pipes)
+* Planned: Streets-wise aggregation option
+
+To answer these questions, at the moment,
+the LP and MILP optimisation library oemof.solph is used.
+Other approaches, e.g. heuristic approaches, might follow.
+
+The following sections will give an overview about the general usage/workflow,
+(the necessary input data, the different optimisation settings and options,
+the results), and second, the underlying mathematical description.
+
+Usage
+-----
+
+Links to the subsections:
+
+* :ref:`Overview`
+* :ref:`Input Data`
+* :ref:`Label systematic <Label system>`
+* :ref:`Results`
+
+.. _Overview:
+
+Overview
+~~~~~~~~
+
+The optimisation of a given *ThermalNetwork* is executed by:
+
+.. code-block:: python
+
+    import dhnx
+
+    tnw = dhnx.network.ThermalNetwork()
+    
+    tnw = network.from_csv_folder('path/to/thermal_network')
+
+    invest_opt = dhnx.input_output.load_invest_options('path/to/invest_options')
+
+    tnw.optimize_investment(invest_options=invest_opt)
+
+For executing an optimisation, you must provide the `invest_options` additional to the previous
+data, which defines a *ThermalNetwork*. Both are explained in the following section.
+
+.. _Input Data:
+
+Input Data
+~~~~~~~~~~
+
+In this section, it is firstly revised, what input data is exactly necessary
+from the *ThemalNetwork* class, and then explained, what data needs to be
+provided as `invest_options` and as `settings`.
+
+The following figure provides an overview of the input data:
+
+.. 	figure:: _static/DHNx_Input_data.svg
+   :width: 100 %
+   :alt: optimization_input_data.svg
+   :align: center
+
+   Fig. 1: Optimisation Input Data
+
+The structure of the input data might look a bit confusing at the beginning, but provides a lot of
+options for building up complex district heating models. There are two groups of data:
+geo-referenced data and parameters for the investment optimisation (with might include other
+oemof-solph related data). All data needs to be provided in csv files. This means the geo-referenced
```

Comment:
This sentence contradicts the schematic above and even more directly, the previous sentence. Proposal: There are two groups of data: Firstly, data that describes the components and the connectivity of the network, required by the ThermalNetwork class. Secondly, data that is necessary for the investment optimization.